src/libs/antares/study/variable-print-info.cpp

```diff
@@ -86,12 +86,6 @@ void variablePrintInfoCollector::add(const AnyString& name,
 // ============================================================
```

Comment:
About **clear()** function : 
Currently, we need a **clear()** in class **AllVariablesPrintInfo** because the vector of **VariablePrintInfo** expects pointers to **VariablePrintInfo**.
Instead of : 
```C++
  std::vector<VariablePrintInfo*> allVarsPrintInfo;
```
we should have : 
```C++
  std::vector<VariablePrintInfo> allVarsPrintInfo;
```
This would allow to remove the **clear()** function and the **destructor** to be a **default** one.